src/parsers/urdf/model.cpp

```diff
@@ -33,13 +33,27 @@ namespace se3
   {
     namespace details
     {
+      void appendBodyToJoint(Model& model, const Model::JointIndex jid,
+                             const boost::shared_ptr< ::urdf::Inertial> Y,
+                             const SE3 & placement,
+                             const std::string & body_name)
```

Comment:
I hesitated between modifying the method `Model::appendBodyToJoint` and creating this function. I think the check for empty inertias would best fit in `Model::appendBodyToJoint` (other parsers would make use of it).